package.json

```diff
@@ -3,9 +3,16 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "bootstrap": "^4.1.1",
+    "enzyme": "^3.3.0",
+    "enzyme-adapter-react-16": "^1.1.1",
+    "jquery": "^3.3.1",
+    "popper.js": "^1.14.3",
```

Comment:
If you want to use popup modal, try to build your own react components your using [react-modal](https://github.com/reactjs/react-modal)